src/ec/packages/ec-component-footer-standardised/footer-standardised.story.js

```diff
@@ -1,29 +1,279 @@
-/* eslint-disable no-param-reassign */
+/* eslint-disable no-param-reassign, no-shadow */
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
+import { withKnobs, button, text } from '@storybook/addon-knobs';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 
-import data from './demo/data';
+import sections from './demo/data';
```

Comment:
why do we differ here from other components should we not have a standerd way of naming our data ?

Reply:
i'd say not at this level, @Joosthe , ok for consistency but it's normal that we create variables for them to mean something for us, this is the simplest use case, you only have one "data" object you get from the specs, but most of the times you have many, what about the consistency then, we name everything data1, data2, data3...? So, you are right in general in looking for consistency, but that doesn't mean we are going to have everything "aligned", stories are "individual" by definition, components are all different, being too strict is not going the help us.

Reply:
I totally agree that we choose variable names that mean something to us. I don't mean strict the sense like data1, data2 , data3 this could be dataPrimary, dataSecondary, dataCall, ... . but i this that we should call these variables always data to begin with and your instance the default data in buttons should always and in data. For example:
`const primaryData = { ...dataPrimary };`. I don't see the added value to call data sections